docs/migrate/migrate-update-parm.rst

```diff
@@ -0,0 +1,115 @@
+Update parameter values of parametrized circuits during algorithm execution
+===========================================================================
+
+We consider an algorithm, such as VQE, that iteratively calls one of the
+primitive programs (here, the Estimator) during its execution.
+
+Overview
+--------
+
+Using the estimator object for the calls allows the cloud backend to
+recognize calls that originate from the same algorithm and to prioritize
+their execution, making the overall algorithm execution time much
+faster.
```

Comment:
This is only true if the jobs run in a session. So perhaps the wording can be something like

```
Initializing an estimator object inside a session allows...
```